eventmodel.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE "event" (
+  "event_id" integer PRIMARY KEY NOT NULL,
+  "title" varchar NOT NULL,
+  "tags" varchar,
+  "description" text,
+  "form_link" varchar NOT NULL,
+  "video_link" varchar NOT NULL,
+  "flyer" varchar,
+  "date" datetime,
+  "location" varchar,
+  "speaker" varchar,
+  "type" varchar,
```

Comment:
tags and types will be redundant 

I suggest creating a tags table and type table (id, value) to avoid the redundancy and ensure similar label usage

@Hkataya what do u think?